bin/update-ts-project-refs.js

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to update TypeScript project references based on
+ * lerna's local package dependencies.
+ *
```

Comment:
+1 to automate this process 👏 

I think we need to run this script every time we add a new inter-repo dependency, is that correct? At minimum, please describe this step in our dev docs at https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md.

I think docs is not enough, we will often forget to run the script. IMO, we should check for consistency between package dependencies and ts project dependencies as part of our Travis CI build, similarly to how we check that package locks don't refer to monorepo packages, etc.

Reply:
It's being run as part of `postinstall` after `lerna bootstrap`.